platform-sdk/swirlds-virtualmap/src/testFixtures/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+module com.swirlds.virtualmap.test.fixtures {
+    exports com.swirlds.virtualmap.test.fixtures;
+
+    requires transitive com.swirlds.common;
+    requires transitive org.junit.jupiter.api;
+    requires com.swirlds.virtualmap;
+    requires java.desktop;
```

Comment:
🤨 created https://github.com/hashgraph/hedera-services/issues/10381

Reply:
Now I was also curios... and fixed it directly. Wrong import as so often 🤷 - https://github.com/hashgraph/hedera-services/pull/8456/commits/d058ac579d66890137ed78b33fba5ef515b576c5